### PR TITLE
Fixed issue

### PR DESCRIPTION
**Pull request by ModEraNoah**

### CODE REVIEW

File: tasks/typescript/easy/index.ts [hunk 1]
```diff
@@ -1,9 +1,9 @@
 // TypeScript - Easy
 
 function greet(name: string): string {
-    // TODO: Add return statement
+  return name;
```
Comment (nikohoffren) [inline tasks/typescript/easy/index.ts:4]: This should return a greeting message "Hello, John Doe!" as it is requested in the issue description. Right now it returns the name John Doe.

File: tasks/typescript/medium/calculator.ts [hunk 2]
```diff
@@ -1,12 +1,31 @@
...
+    return a * b;
+  }
+
+  export function divide(a: number, b: number) {
+    if (b === 0) {
+      console.log("Ups.... tried to divide by 0");
+      return null;
+    }
+    return a / b;
+  }
```
Comment (nikohoffren) [inline tasks/typescript/medium/calculator.ts:23]: Code for the calculator looks good!

**Review by nikohoffren — CHANGES_REQUESTED** [review]
I can merge this after the TypeScript easy return statement is fixed.

**Comment by ModEraNoah** [comment]
Yeah, thanks for the hint. I will also do the "Hard" right now so please feel free to wait with the merge

**Comment by nikohoffren** [comment]
> Yeah, thanks for the hint. I will also do the "Hard" right now so please feel free to wait with the merge
> 

Sure! You can also make separate pull requests for every task so you can get more merged PR:s but this is fine too.

**Comment by ModEraNoah** [comment]
Took a little bit longer than expected but it should now work fine.
PS: In my opinion this repository and all the work you are willing to take for it is really great and I wanted to let you know that I really appreciate this possibility to get a first glance into programming and public VCS-work

**Review by nikohoffren — APPROVED** [review]
Code for all of the tasks looks good!

**Comment by nikohoffren** [comment]
## Merged

> This is an automated message from Fork, Commit, Merge [BOT].

Thank you for your contribution! Your pull request has been merged. The files have been reset for the next contributor.

### What's next?

If you're looking for more ways to contribute, I invite you to check out my other projects. Just click [here](https://github.com/nikohoffren?tab=repositories) to find more. These projects contain real issues that you can help resolve. You can also check out the [Influences](https://github.com/nikohoffren/fork-commit-merge#influences) section in the README to find more projects similar to this one.

Also please leave a star to this project if you feel it helped you, i would really appreciate it.

I look forward to seeing your contributions!

**Pull request merged**